app/templates/gitignore

```diff
@@ -1,3 +1,6 @@
 node_modules<% if (nyc) { %>
 .nyc_output
 coverage<% } %>
+yarn.lock
```

Comment:
Even if I don't like `yarn`, I think it should be added so `yarn` users can contribute without switching tools or manually discard there lockfile